cpp/src/io/csv/csv_reader.cu

```diff
@@ -537,8 +537,8 @@ gdf_error read_csv(csv_read_arg *args)
 	//-----------------------------------------------------------------------------
 	//--- Auto detect types of the vectors
 
-	// if(args->dtype==NULL){
-	if(args->names==NULL){
+	if(args->dtype==NULL){
+	//if(args->names==NULL){
```

Comment:
Please remove commented-out code. If it is needed for the future, add a comment explaining what it is waiting on. (If it's just posterity, remove it -- the git history is posterity 😄  ).

Reply:
removed